todos-app/backend/task_service/spec/models/task_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
```

Comment:
the way you wrote the spec in [request](todos-app/backend/task_service/spec/requests/task_spec.rb) is so nice!

it is related to this article 
https://www.betterspecs.org/

according to this document, this spec code could be rewritten to look something like this I think.

```
context 'when status is not in 0, 1, 2'
  it 'is invalid'
    subject.status = 3
    expect(subject).to_not be_valid
  end
end
```